server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -1620,7 +1620,9 @@ protected void runInContext() {
                 for (StoragePoolVO pool : pools) {
                     List<VolumeVO> volumes = _volsDao.findByPoolId(pool.getId(), null);
                     for (VolumeVO volume : volumes) {
-                        if (volume.getFormat() != ImageFormat.QCOW2 && volume.getFormat() != ImageFormat.VHD && volume.getFormat() != ImageFormat.OVA && (volume.getFormat() != ImageFormat.RAW || pool.getPoolType() != Storage.StoragePoolType.PowerFlex)) {
+                       if (volume.getFormat() != ImageFormat.QCOW2 && volume.getFormat() != ImageFormat.VHD &&
+                            volume.getFormat() != ImageFormat.OVA && pool.getPoolType() != Storage.StoragePoolType.PowerFlex &&
+                            pool.getPoolType() != Storage.StoragePoolType.FiberChannel) {
```

Comment:
Only if its FiberChannel or PowerFlex.   I simply extended what was done for Powerflex in this case.  The stats data is available from the storage provider in the case of Primera and Pure.  I suspect its available for other types of storage providers.  This code probably should be refactored to use the storage provider in some way instead of trying to determine the information by looking at the format, but I did not want to get into a more extension change at this time to try to do that.